keras_nlp/models/falcon/falcon_backbone.py

```diff
@@ -0,0 +1,243 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Falcon backbone model."""
+
+import copy
+
+from tensorflow import keras
+from tensorflow.experimental import dtensor
+from tensorflow.experimental.dtensor import Layout
+
+from keras_nlp.api_export import keras_nlp_export
+from keras_nlp.layers.token_and_position_embedding import (
+    TokenAndPositionEmbedding,
+)
+from keras_nlp.layers.transformer_decoder import TransformerDecoder
+from keras_nlp.models.backbone import Backbone
+from keras_nlp.models.falcon.falcon_presets import backbone_presets
+from keras_nlp.utils.python_utils import classproperty
+
+
+def Falcon_kernel_initializer(stddev=0.02):
```

Comment:
`falcon_kernel_initializer`

Reply:
Make sure to check all these with upstream too. What are the default parameter initializations they use, when starting from scratch for pre-training?

Do you have a link handy to where falcon original/hf code does this?